zocalo/cli/go.py

```diff
@@ -37,7 +37,8 @@ def run():
         metavar="RCP",
         action="append",
         default=[],
-        help="Name of a recipe to run. Can be used multiple times. Recipe names correspond to filenames (excluding .json) in /dls_sw/apps/zocalo/live/recipes",
+        help="Name of a recipe to run. Can be used multiple times. Recipe names correspond to filenames (excluding .json)."
+        " This directory is defined in the _recipe_basepath attribute of Dispatcher services.",
```

Comment:
assert this information is not helpful for anyone running `zocalo.go --help`

Reply:
Okay, suggestions?

Reply:
Presumably go could be run without having this defined if you only used -f ? Currently the message is only helpful if you use DLSDispatcher at Diamond